static/templates/stream_settings/stream_settings.hbs

```diff
@@ -75,43 +75,39 @@
                 <div id="stream_change_property_status{{sub.stream_id}}" class="stream_change_property_status alert-notification"></div>
             </div>
             <div class="subscription-config">
-                <ul>
+                <div class="other_settings">
```

Comment:
I think the elements this applies to are the `subsection-parent` or `subsection-header` classes from the Settings UI; it might make sense to make the two layers here match those names (also might give better names for these divs in the CSS that are more declarative).